blog/_posts/2017-09-08-alvida-rgsoc-2017.md

```diff
@@ -0,0 +1,93 @@
+On our third meeting with Kelsey, she had said that "It is better to over-communicate rather than Under-communicate" and thus, we would ( shamelessly 🙈 ) ping them on slack or on the issues whenever we were stuck and a descriptive solution to our questions would follow. Many a times, Kelsey would explain us our doubts on our call. Despite whatever mistakes we would make, the committee members were always positive and ensured that we were able to rectify our errors and learn from them. Many a times, they've tagged us in several issues that they thought we should be aware of and sent us links and resources to escalate our learning.
+
+![Team 276linesofCode](/img/blog/2017/2017-09-08-comment.png)
+<div class="image-credits"><b>@HipsterBrown, an active Tessel Steering Committee member, responding to us :) (Credits - Slack) </b></div>
+
+Tessel has provided us amazing Mentorship, one that is really hard to find. We are so glad that we began our journey with Open Source with such a welcoming community, so much so that we aim to work for Tessel even after RGSoC is over. Tessel, We love you!
+
+## Extras
+
+In these three months, we not only focussed on core development work but we also went on presenting a few lightning talks. We gave two of them - one for Women Who Code, Delhi and another for LinuxChix India. In the WWC-D Meetup, we introduced the participants to Computer Vision in Python. It was a great experience, since, we got to delve deeper into Open CV, explain the concepts involved to the attendees, cleared their doubts. The audience was very responsive and we even got a positive feedback in the end. So, through this we were able to spread our knowledge about Computer Vision to a few enthusiasts beginning their journey in this. We introduced RGSoC in the beginning of our presentation, making the audience aware about what it is and how it really works.
+
+![Team 276linesofCode](/img/blog/2017/2017-09-08-wwcdtalk.png)
+<div class="image-credits"><b>Women Who Code Delhi publishes a review of our talk (Credits - Twitter) </b></div>
+
+
+The LinuxChix India Meetup was focussed more on Open Source Contributions, RGSoC and Tessel. Our supervisor Vaishali had introduced us to the community and put us in touch with the organizer of the meetup. We spoke about RGSoC - How to apply, what all do we do, our work, our social media, our blog posts, in and all everything. After that, we went on to demonstrate our project - Tessel. There were several Open Source enthusiasts who had loads of questions about Tessel and we were more than delighted to answer them. One person even wanted to use Tessel in his project that he was thinking of starting!
```

Comment:
Do you have slides of your presentations somewhere online? It would be good to attach them somewhere here. If not, then it's fine. :)

Reply:
Yes we do have slides for both of them. I'll just add the links